.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+# BUG Template
+
+## Expected Behaviour
```

Comment:
there is a LOT here. I worry that this might be off-putting, particularly if someone thought they had to fill all of it out.

This is what's currently set via the issues link in bower.json:
> please provide a full description of the problem, including steps on how to replicate, what browser(s)/device(s) the problem occurs on and, where helpful, screenshots.

Probably could use something about making sure you supply the framework and - where appropriate - plugin version. And to state whether you're using framework or AT.

Reply:
I've found just listing what you want leads people to ignore it unless you really spell it out for people they will just keep creating one line issues that mean you have to go back and forth and few times to get the info out of them.

We have found on the AT that people are filling it out and its been really helpful when trying to debug to just have a list of reproduction steps along with the node version etc.

Reply:
No sense in learning the same lesson twice.

Reply:
@moloko shall I revert to a form? If so are there any changes you'd want?

---
### 1. Problem found or the feature required


### 2. Steps to reproduce (if applicable)
1. 
1. 
1. 

### 3. Environment (delete as appropriate)
  - Authoring tool version: 
  - Framework version: 
  - Plugin version: 
  - Device(s):
  - Operating System(s): 
  - Browser(s): 

### 4. Screenshots (if you need can)

---

Reply:
@oliverfoster Yeah, I missed the above comments while reviewing. The above template would be more appropriate to log an issue.